src/docfx/lib/markdown/MarkdownUtility.cs

```diff
@@ -143,14 +145,19 @@ private static MarkdownPipeline CreateTocMarkdownPipeline()
             return builder.Build();
         }
 
+        private static Config GetConfig()
+        {
+            return t_status.Value.Peek().Context.Config;
```

Comment:
to get the actual path of rule file, use `context.RestoreFileMap.GetRestoredFilePath`

Reply:
sure